app_mobilenet_alpha_1/src/support.cpp

```diff
@@ -0,0 +1,70 @@
+#include <platform.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <xcore/channel.h>
+
+#include "xcore_optimised_mobilenetV2_input_160x160x3_alpha_1.tflite.h"
+
+#if defined __GNUC__
```

Comment:
Remove lines 8 to 18. They are not being used in this case.

Reply:
Or could leave it in here but add in a comment on line 17 mentioning that the tensor arena size specified here must be the largest of all shared arenas.